<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>2005</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Progress Viewer</string>
  </property>
  <widget class="QPushButton" name="executePB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>260</width>
     <height>85</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Execution process will take some time. And also it open your browser, so please don't interfere in the browser process.
 Do you want to continue?</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="closePB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>133</y>
     <width>270</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>135</y>
     <width>260</width>
     <height>40</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Wait! Process is ongoing...
</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressPB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>185</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pausePB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel URL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save and Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopPB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
